runtime/src/lib.rs

```diff
@@ -235,46 +211,27 @@ impl frame_system::Config for Runtime {
 	type MaxConsumers = ConstU32<16>;
 }
 
-impl pallet_randomness_collective_flip::Config for Runtime {}
-
 parameter_types! {
-	pub const MaxAuthorities: u32 = 32;
+	pub const UncleGenerations: u32 = 0;
 }
 
-impl pallet_aura::Config for Runtime {
-	type AuthorityId = AuraId;
-	type DisabledValidators = ();
-	type MaxAuthorities = MaxAuthorities;
+impl pallet_authorship::Config for Runtime {
+	type FindAuthor = ();
+	type UncleGenerations = UncleGenerations;
+	type FilterUncle = ();
+	type EventHandler = ();
 }
 
-impl pallet_grandpa::Config for Runtime {
-	type Event = Event;
-	type Call = Call;
-
-	type KeyOwnerProofSystem = ();
-
-	type KeyOwnerProof =
-		<Self::KeyOwnerProofSystem as KeyOwnerProofSystem<(KeyTypeId, GrandpaId)>>::Proof;
-
-	type KeyOwnerIdentification = <Self::KeyOwnerProofSystem as KeyOwnerProofSystem<(
-		KeyTypeId,
-		GrandpaId,
-	)>>::IdentificationTuple;
-
-	type HandleEquivocation = ();
-
-	type WeightInfo = ();
-	type MaxAuthorities = MaxAuthorities;
-}
+impl pallet_randomness_collective_flip::Config for Runtime {}
 
 parameter_types! {
-	pub const MinimumPeriod: u64 = SLOT_DURATION / 2;
+	pub const MinimumPeriod: u64 = 5;
```

Comment:
Not sure we really need a minimum period here. Do you know if we can simply set this to 0?

Reply:
MinimumPeriod is The minimum period between blocks production in milliseconds. 
But it should not be set as 0 as it is used by set function (set time) in pallet timestamp and invoked once per block and will prevent the block from finalization.
[see pallet-timestamp](https://github.com/paritytech/substrate/blob/96738a1f3dbe4a1fb6abb385920fb403ee961488/frame/timestamp/src/lib.rs#L186)
As for setting it to 1 made zero difference on my test so I would keep 5 as a minimum secure threshold

Reply:
Ah, I didn't read the docs carefully enough - I just assumed it was in seconds. Five milliseconds sounds totally reasonable to me